### PR TITLE
Highlight partial AC testcases differently

### PR DESCRIPTION
**Pull request by Xyene**
Supersedes https://github.com/DMOJ/online-judge/pull/1802, fixes https://github.com/DMOJ/online-judge/issues/1572.

image

### CODE REVIEW

File: resources/submission.scss [hunk 1]
```diff
@@ -288,7 +288,7 @@ label[for="language"], label[for="status"] {
 }
 
 .case-_AC {
```
Comment (quantum5) [inline resources/submission.scss:290]: Why did this even exist in the first place?

**Review by quantum5 — APPROVED** [review]

**Comment by quantum5** [comment]
Actually the colour may be a bit hard to see on a white background.

**Comment by kiritofeng** [comment]
image
Agree with @quantum5, maybe we should pick something closer to the full AC colour

**Comment by Xyene** [comment]
You're late, I've already updated the PR with a @quantum5-approved color.

**Pull request merged**